pkg/workflow_execution.go

```diff
@@ -824,8 +824,7 @@ func (c *Client) GetWorkflowExecutionLogs(namespace, uid, podName, containerName
 			return nil, util.NewUserError(codes.InvalidArgument, "Invaild range.")
 		}
 		opts.SetRange(0, int64(endOffset))
-
-		stream, err = s3Client.GetObject(config.ArtifactRepository.S3.Bucket, "artifacts/"+namespace+"/"+uid+"/"+podName+"/"+containerName+".log", opts)
+		stream, err = s3Client.GetObject(config.ArtifactRepository.S3.Bucket, config.ArtifactRepository.S3.KeyFormat+"/"+containerName+".log", opts)
```

Comment:
@Vafilor this won't work as the tokens in `config.ArtifactRepository.S3.KeyFormat` need to be replaced with the actual values.

Reply:
@rushtehrani Updated.